src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIOptions.cs

```diff
@@ -63,6 +63,18 @@ public class SwaggerUIOptions
         /// Gets or sets the path or URL to the Swagger UI CSS file.
         /// </summary>
         public string StylesPath { get; set; } = "./swagger-ui.css";
+
+        /// <summary>
+        /// Expose the <c><see cref="SwaggerUIOptions.ConfigObject">ConfigObject</see>.Urls</c> object via a new
+        /// "&lt;<see cref="SwaggerUIOptions.RoutePrefix"/>&gt;/&lt;<see cref="SwaggerUIOptions.SwaggerDocumentUrlsPath"/>&gt;
+        /// route so that external code can auto-discover all swagger documents.
```

Comment:
```suggestion
        /// Gets or sets whether to expose the <c><see cref="SwaggerUIOptions.ConfigObject">ConfigObject</see>.Urls</c> object via an
        /// HTTP endpoint with the URL specified by <see cref="SwaggerDocumentUrlsPath"/>
        /// so that external code can auto-discover all Swagger documents.
```

Reply:
Fixed